.github/workflows/tests.yml

```
@@ -0,0 +1,31 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Run Unit Tests
+    runs-on: ubuntu-latest
+    container:
+      image: busybox
+      options: --user root
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```
Use latest checkout action